src/Simplex_tree/benchmark/random_simplices.h

```diff
@@ -0,0 +1,46 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2025 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <iostream>
+#include <random>
+#include <numeric>  // for std::iota
+#include <algorithm>  // for std::shuffle
+#include <vector>
+
+std::random_device rd;
+
+/** \brief Returns a random simplex.
```

Comment:
I was afraid doxygen might try to show it, but I don't see it in the generated doc, that's good.